PR #81 - aidl hal version specification

Ulrond opened the pull request:
Added baseline documentation for review.

Ulrond committed:
Added #81 - aidl hal version specification

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/rdkcentral/rdk-halif-aidl?pullRequest=82) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/rdkcentral/rdk-halif-aidl?pullRequest=82) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: Ulrond<br/>:x: Aniket Patil<br/><hr/>**Aniket Patil** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/rdkcentral/rdk-halif-aidl?pullRequest=82) it.</sub>

aniketp1809 committed:
Task: AIDL HAL Versioning Specification

Updated AIDL HAL Versioning specifications

Reviewed by: Gerald Weatherup, Varghese Mathai
Issue URL: https://github.com/rdkcentral/rdk-halif-aidl/issues/81

aniketp1809 committed:
Task: AIDL HAL Versioning Specification

Updated as per the new directory structure to be followed.

Reviewed by: Gerald Weatherup, Varghese Mathai
Issue URL: https://github.com/rdkcentral/rdk-halif-aidl/issues/81

aniketp1809 committed:
Task: AIDL HAL Versioning Specification

Updated document as per recent changes in the versioning framkework

Reviewed by: Gerald Weatherup, Varghese Mathai
Issue URL: https://github.com/rdkcentral/rdk-halif-aidl/issues/81

kanjoe24 committed:
Merge branch 'develop' into feature/81-task-aidl-hal-versioning-specification

Ulrond commented:
@kanjoe24  let me bring this back to me, I'll need to re-sync with branch 75 requirements.

Ulrond commented:
Needs to be re-visited against the new implementation

kanjoe24 commented:
this will be upgraded as part of #75
